librz/analysis/analysis.c

```diff
@@ -103,8 +103,8 @@ RZ_API RzAnalysis *rz_analysis_new(void) {
 	analysis->diff_thbb = RZ_ANALYSIS_THRESHOLDBB;
 	analysis->diff_thfcn = RZ_ANALYSIS_THRESHOLDFCN;
 	analysis->syscall = rz_syscall_new();
-	analysis->arch_target = rz_arch_target_new();
-	analysis->platform_target = rz_arch_platform_target_new();
+	analysis->arch_target = rz_platform_target_new();
+	analysis->platform_target = rz_platform_target_index_new();
```

Comment:
What's the diff between these? Hard to tell from the names

Reply:
dunno, cannot find a better name since one structure is just a map + filename (rz_platform_target_index) and the other one has details regarding the platform, like cpu stuff, mmio, etc..